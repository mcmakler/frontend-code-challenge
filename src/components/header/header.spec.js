import expect from 'expect';
import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Header from './header';

describe('Header specs', () => {
  it('should match snapshot', () => {
    const component = ReactTestRenderer.create(<Header />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
