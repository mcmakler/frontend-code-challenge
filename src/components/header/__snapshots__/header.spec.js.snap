// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header specs should match snapshot 1`] = `
<div
  className="header"
>
  <h1
    className="header__title"
  >
    Vacant Apartments
  </h1>
</div>
`;
